api/sheets/index.md

```diff
@@ -8,38 +8,58 @@ has_children: true
 
 # Data Commons Sheets API
 
-The **Data Commons Sheets API** is a Google Sheets add-on that enables Google Sheets
-users to import data from the Data Commons knowledge graph. The add-on provides an
-interface for finding a location's [DCID](/glossary.html), as well as some custom functions for
-importing data to a spreadsheet.
+The **Data Commons Sheets API** is a Google Sheets add-on that enables Google Sheets users to import data from the Data Commons knowledge graph. The add-on provides an interface for finding a location’s unique Data Commons identifier ([DCID](glossary.html)), and some custom functions for importing data into a spreadsheet.
 
-Also read our [step-by-step guides](tutorials/) for more examples.
+> **NOTE:**
+> 
+> For more information on DCIDs, refer to [Unique Identifiers (dcid)](https://docs.datacommons.org/bigquery/unique_identifiers.html).
 
-## Getting started
+Also read our [step-by-step guides](tutorials/) for more examples.
 
-Install the add-on from [G Suite Marketplace](https://gsuite.google.com/marketplace/app/data_commons/454343067575). To enable the add-on in a document, click "Add-ons > Data Commons > Fill place dcids". You may use the resulting sidebar to start [finding dcids](/api/sheets/get_dcid.html) in the United States, or close it and reopen it at any time. Note that none of the custom functions will work in a given document until you have enabled the add-on by choosing "Fill place dcids".
+## Installing and Enabling the Sheets Add-On
 
-![](/assets/images/sheets/sheets_menu_bar.png)
+Install and enable the Sheets add-on from Google Workspace Marketplace, as follows:
 
-## Finding a place's DCID
+1. Install the Sheets add-on by clicking **[Google Workspace Marketplace](https://gsuite.google.com/marketplace/app/data_commons/454343067575)** .
+2. Enable the Sheets add-on by clicking the Google Sheets **Extensions** menu, pointing to **Data Commons**, and then by clicking **Fill place dcids**.
+3. Use the resulting sidebar that appears on the right side of the page to start [finding DCIDs](https://docs.datacommons.org/api/sheets/get_name.html) in the United States. You can close and reopen the sidebar at any time.
```

Comment:
please use a relative path (i.e. drop https://docs.datacommons.org from the URL). this comment applies to all URLs in these PRs

Reply:
Ok